packages/react/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  rules: {
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-unused-vars': 'off',
+    'react/prop-types': 'off',
+    'react/no-find-dom-node': 'off',
+    'no-undef': 'off'
```

Comment:
I think we should keep these on and disable the rules in the places we need them disabled.